docs/usage.md

```diff
@@ -57,3 +58,14 @@ The baremetal operations listed here are not available to non-administrators by
 * **Power on assigned node:** `openstack baremetal node power on <node-id-or-name>` [[full reference](https://docs.openstack.org/python-ironicclient/latest/cli/osc/v1/index.html#baremetal-node-power-on)]
 * **Power off assigned node:** `openstack baremetal node power off <node-id-or-name>` [[full reference](https://docs.openstack.org/python-ironicclient/latest/cli/osc/v1/index.html#baremetal-node-power-off)]
 * **Reboot assigned node:** `openstack baremetal node reboot <node-id-or-name>` [[full reference](https://docs.openstack.org/python-ironicclient/latest/cli/osc/v1/index.html#baremetal-node-reboot)]
+
+## <a name="network"></a>Network
+
+### Administrator/User
+
+* **Create network:** `openstack network create <options>` [[full reference](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/network.html#network-create)]
+  * Please refer to the linked CLI reference for the full list of options.
+* **Create subnet:** `openstack subnet create <options>` [[full reference](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/subnet.html#subnet-create)]
```

Comment:
I would expand this slightly since the `network` argument is required: `openstack subnet create --network <network> [<options>] <name>`

Reply:
Ah, good idea - fixed!